src/simplewallet/simplewallet.cpp

```diff
@@ -5608,7 +5608,7 @@ bool simple_wallet::register_service_node(const std::vector<std::string> &args_)
   std::vector<uint64_t> portions;
   uint64_t portions_for_operator;
   bool autostake;
-  if (!service_nodes::convert_registration_args(m_wallet->nettype(), address_portions_args, addresses, portions, portions_for_operator, autostake))
+  if (!service_nodes::convert_registration_args(m_wallet->nettype(), address_portions_args, addresses, portions, portions_for_operator, autostake, boost::none))
```

Comment:
It would be really useful to print the error here. There's been many times where people have problems registering in the community and it's hard to figure out if its a problem with their copy pasting or a bug on our side.